src/library/Uno.Material/Styles/Controls/v2/Slider.xaml

```diff
@@ -42,36 +77,36 @@
 					<Border Background="{TemplateBinding Background}"
 					        BorderBrush="{TemplateBinding BorderBrush}"
 					        BorderThickness="{TemplateBinding BorderThickness}"
-					        CornerRadius="10" />
+					        CornerRadius="{ThemeResource SliderThumbCornerRadius}" />
 				</ControlTemplate>
 			</Setter.Value>
 		</Setter>
 	</Style>
 
 	<!--  Material Slider Style  -->
 	<Style x:Key="MaterialSliderStyle"
-	       TargetType="Slider">
+		   TargetType="Slider">
 
 		<Setter Property="Background"
-		        Value="{ThemeResource MaterialSliderTrackBrush}" />
+				Value="{ThemeResource SliderTrackFill}" />
 		<Setter Property="Foreground"
-		        Value="{ThemeResource PrimaryBrush}" />
+				Value="{ThemeResource SliderTrackValueFill}" />
 		<Setter Property="BorderThickness"
-		        Value="0" />
+				Value="0" />
 		<Setter Property="FontFamily"
-		        Value="{ThemeResource MaterialRegularFontFamily}" />
+				Value="{ThemeResource MaterialRegularFontFamily}" />
 		<Setter Property="FontSize"
-		        Value="{ThemeResource LabelMediumFontSize}" />
+				Value="{ThemeResource LabelMediumFontSize}" />
```

Comment:
Let's extract these typography values into the ThemeDictionaries are themed resources. Something like `SliderFontFamily` and `SliderFontSize`